cluster-autoscaler/Makefile

```diff
@@ -64,15 +64,9 @@ dev-release-arch-%: build-arch-% make-image-arch-% push-image-arch-%
 make-image: make-image-arch-$(GOARCH)
 
 make-image-arch-%:
-ifdef BASEIMAGE
-	docker build --pull --build-arg BASEIMAGE=${BASEIMAGE} \
+	$(ENVVAR) GOOS=$(GOOS) GOARCH=$* docker build --pull \
```

Comment:
I think you can remove `$(ENVVAR)` here since we replaced it with the hardcoded version in the dockerfile.

For consolidating all 3 Dockerfiles into 1, you'll need to also pass `--platform=linux/$*` here. Similar to https://github.com/kubernetes/autoscaler/blob/e5307467a7d1dea5bc8d9f6e5939214ae36d42a2/addon-resizer/Makefile#L58.

Reply:
both done